src/Index/Index.php

```diff
@@ -15,14 +15,31 @@ class Index implements ReadableIndex, \Serializable
     use EmitterTrait;
 
     /**
-     * An associative array that maps fully qualified symbol names to Definitions
+     * An associative array that maps namespaces to
+     *     an associative array that maps fully qualified symbol names
+     *     to global Definitions, e.g. :
+     *     [
+     *         'Psr\Log\LoggerInterface' => [
```

Comment:
Well, the thing is that everywhere in the app we're using `$fqn` and referring to `fully qualified namespace`, but we're using it for all symbols, ie `Psr\Log\LoggerInterface->log()` and also `Psr\Log\LoggerInterface`.

Which term could describe `Psr\Log\LoggerInterface` ? The `left hand side of access operator` ? Quite long for a name 😝 . What's your opinion on this point ?